recipe/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd src
+
+TCL_VERSION="$(echo 'puts [info tclversion]' | tclsh)"
+TCL_LIB_PATH="$PREFIX"/lib/tcl"$TCL_VERSION"
+BWIDGET_PATH="$TCL_LIB_PATH"/bwidget
+
+[[ ! -d "$BWIDGET_PATH" ]] || exit 1
+mkdir -p "$BWIDGET_PATH"
+cp -r * "$BWIDGET_PATH"
```

Comment:
```suggestion
mv * "$BWIDGET_PATH"/
```
Doing a move instead of a copy will save disk on the build workers.

Reply:
Fixed